app/assets/javascripts/ui_components/active_buttons.coffee

```diff
@@ -0,0 +1,5 @@
+$ ->
+  $('.btn-group .btn').click ->
+    if $(this).siblings().hasClass 'active'
```

Comment:
Is this check necessary? I think, it wouldn't fail if you just execute it anyway